### PR TITLE
codesmell fix for PRL-4525

### PR DESCRIPTION
**Pull request by BharadwajSairam**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-4525

### Change description ###

Tech debt: java:S2699, S1874,S1168, S112 & S5786

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by shashisk97 — APPROVED** [review]
Approved

File: src/main/java/uk/gov/hmcts/reform/prl/services/courtnav/CourtNavCaseService.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public class CourtNavCaseService {
     private final DocumentGenService documentGenService;
     private final AllTabServiceImpl allTabService;
 
-    public CaseDetails createCourtNavCase(String authToken, CaseData caseData) throws Exception {
+    public CaseDetails createCourtNavCase(String authToken, CaseData caseData) {
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/services/courtnav/CourtNavCaseService.java:63]: good one. its covered in another PR too.

File: src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java [hunk 2]
```diff
@@ -86,7 +80,7 @@ private Map<String, Object> getAdditionalData(String caseId, String letterType,
         return additionalData;
     }
 
-    private byte[] getDocumentsAsBytes(String docUrl, String authToken, String s2sToken) throws IOException {
+    private byte[] getDocumentsAsBytes(String docUrl, String authToken, String s2sToken) {
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java:83]: same covered in another PR.

File: src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java [hunk 3]
```diff
@@ -48,13 +48,7 @@ public UUID send(String caseId, String userToken, String letterType, List<Docume
...
         String s2sToken = authTokenGenerator.generate();
         final List<String> stringifiedDocuments = documents.stream()
-            .map(docInfo -> {
-                try {
-                    return getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken);
-                } catch (IOException e) {
-                    throw new RuntimeException(e);
-                }
-            })
+            .map(docInfo -> getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken))
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java:51]: same question as this - https://github.com/hmcts/prl-cos-api/pull/1813#discussion_r1373908029

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**